lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_completecodefreeze_prechecks.rb

```diff
@@ -0,0 +1,67 @@
+module Fastlane
+    module Actions
+      class AndroidCompletecodefreezePrechecksAction < Action
+        def self.run(params)
+          UI.message "Skip confirm: #{params[:skip_confirm]}"
+          
+          require_relative '../../helper/android/android_version_helper.rb'
+          require_relative '../../helper/android/android_git_helper.rb'
+  
+          UI.user_error!("This is not a release branch. Abort.") unless other_action.git_branch.start_with?("release/")
+  
+          version = Fastlane::Helpers::AndroidVersionHelper::get_public_version
+          message = "Completing code freeze for: #{version}\n"
+          if (!params[:skip_confirm])
+            if (!UI.confirm("#{message}Do you want to continue?"))
```

Comment:
totally codestyle nitpick but Ruby doesn't require parentheses around `if` condtions, so feels a bit strange to read here 😅  Also, you should prefer `unless x` to `if !x`.

```suggestion
          unless params[:skip_confirm]
            unless UI.confirm("#{message}Do you want to continue?")
```

Reply:
I've been meaning to add Rubocop on this repo. There are a lot of Ruby specific syntax and style optimization that I didn't know about which Rubocop taught me.